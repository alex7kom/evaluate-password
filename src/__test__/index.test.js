/* eslint-env jest */
'use strict';

var evaluatePassword = require('../');

var passwords = require('./passwords');

test.each(passwords)('%s', password => {
  expect(evaluatePassword(password)).toMatchSnapshot();
});
