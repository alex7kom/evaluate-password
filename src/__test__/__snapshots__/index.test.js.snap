// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `0`;

exports[`123ABCabc 1`] = `11`;

exports[`1111 1`] = `0`;

exports[`an example of somewhat short passphrase 1`] = `64`;

exports[`j5JSxDUBBkYz!D1$I& 1`] = `39`;

exports[`n0sNsF0grZepgpfs 1`] = `30`;

exports[`password 1`] = `14`;
